src/xpk/core/system_characteristics.py

```diff
@@ -1081,6 +1081,15 @@ def get_system_characteristics(
         'v5p-17920',
     ),
     # v5litepod
+    'v5litepod-8': SystemCharacteristics(
+        '2x4',
+        4,
```

Comment:
- Isn't `ct5lp-hightpu-8t` a SingleHost slice with 1 vm per slice? ([ref](https://cloud.google.com/kubernetes-engine/docs/concepts/plan-tpus#topology))
- Also, [here](https://cloud.google.com/kubernetes-engine/docs/how-to/tpus#create-node-pool) creating a Single-host TPU Slice is explained separately from Multihost. Please make sure xpk node-pool creation/workload flows is compatible with your new SingleHost SystemCharasteristic.

[More info on Single/Multi Host](https://cloud.google.com/tpu/docs/tpus-in-gke)